app/assets/javascripts/course/lectures.js

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
@xyc0562 We are moving js files to `client/bundles/`, you should move this file there and use ES6 syntax.